mesh/mesh.hpp

```diff
@@ -1227,7 +1227,7 @@ class Mesh
        satisfy: v0 < min(v1, v2).
 
        @note Refinement does not work after a call to this method! */
-   virtual void ReorientTetMesh();
+   // virtual void ReorientTetMesh();
```

Comment:
Should we remove that or mark `ReorientTetMesh` as deprecated?

Removing it is fine, but than we should mention that as an "API change" in `CHANGELOG` since some users may very well have that function in their code.

Reply:
I think we should keep it, marked as deprecated. We can probably make it a no-op too but that may have unexpected results -- I'm not sure.

Reply:
If we decide to keep it, I'd prefer not to change the implementation.